README.md

```diff
@@ -36,6 +45,20 @@ Return to your original locations.  In this step all groups will act like Design
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+1. Every time a player scores, a lucky fan from the fans who have signed up on the app and are watching the game are selected. Our services uses the data from the maps on their smartphone to locate them. Then, our services teleports the player who scored into the location that the fan is in, so they can celebrate together for a very short amount of time. Then, the player is teleported back.
+  1. Strengths:
+    1. Will definitely have a positive impact on fans
+    1. Encourage fans to watch the games and sign up
+  1. Weaknesses:
+    1. Could be bad for player morale if they are always being taken out of the game
+    1. Could slow down game play a locate
+    1. Players could become deformed due to constant teleportation
+1. Every time a player scores, a fan or fans from the list are chosen to watch the game through virtual reality. Each fan, when they sign up for the app, is given a virtual reality headset. When they are chosen, they put the headset on and get to see and experience the game from the player's perspective and act like the player for an extended period of time.
```

Comment:
I like this second idea better than the first, mostly because of the feasibility but also because fans would be able to live out basketball games they would normally just watch on TV.